pkgs/tools/typesetting/satysfi/default.nix

```diff
@@ -51,23 +52,24 @@ in
       $out/share/satysfi
     '';
 
-    DUNE_PROFILE = "release";
+    duneVersion = "3";
 
-    nativeBuildInputs = [ ruby dune_3 ];
```

Comment:
ruby is gone here https://github.com/gfngfn/SATySFi/pull/142 in previous version.